ts/screens/profile/FiscalCodeScreen.tsx

```diff
@@ -0,0 +1,156 @@
+    };
+  }
+  public render() {
+    return (
+      <React.Fragment>
+        <DarkLayout
+          allowGoBack={true}
+          headerBody={
+            <Text white={true}>{I18n.t("profile.fiscalCode.title")}</Text>
+          }
+          contentStyle={styles.darkBg}
+          contextualHelp={{
+            title: I18n.t("profile.fiscalCode.title"),
+            body: () => <Markdown>{I18n.t("profile.fiscalCode.help")}</Markdown>
+          }}
+          hideHeader={true}
+          topContent={
+            <React.Fragment>
+              <View spacer={true} />
+              <H5 style={styles.white}>
+                {I18n.t("profile.fiscalCode.fiscalCode")}
+              </H5>
+              <View spacer={true} />
+            </React.Fragment>
+          }
+        >
+          {this.props.profile && (
+            <React.Fragment>
+              <ScrollView
+                horizontal={true}
+                showsHorizontalScrollIndicator={false}
+              >
+                <View style={styles.largeSpacer} />
+                <TouchableOpacity
+                  onPress={() =>
+                    this.setState({
+                      showAsLandscape: true,
+                      showBackSide: false
+                    })
+                  }
+                >
+                  <View style={styles.shadow}>
+                    <FiscalCodeComponent
+                      type={"Full"}
+                      profile={this.props.profile}
+                      getBackSide={false}
+                    />
+                  </View>
+                </TouchableOpacity>
+
+                <View style={styles.spacer} />
+
+                <TouchableOpacity
+                  onPress={() =>
+                    this.setState({
+                      showAsLandscape: true,
+                      showBackSide: true
+                    })
+                  }
+                >
+                  <View style={styles.shadow}>
+                    <FiscalCodeComponent
+                      type={"Full"}
+                      profile={this.props.profile}
+                      getBackSide={true}
+                    />
+                  </View>
+                </TouchableOpacity>
+
+                <View style={styles.largeSpacer} />
+              </ScrollView>
+              <Text white={true} style={styles.text}>
+                {I18n.t("profile.fiscalCode.content")}
+              </Text>
+            </React.Fragment>
+          )}
+        </DarkLayout>
+
+        {this.props.profile && (
+          <FiscalCodeLandscapeModal
+            isVisible={this.state.showAsLandscape}
+            onClose={() => this.setState({ showAsLandscape: false })}
+            showBackSide={this.state.showBackSide}
+            profile={this.props.profile}
+          />
+        )}
+      </React.Fragment>
+    );
+  }
+}
+
+const mapStateToProps = (state: GlobalState) => ({
+  profile: getOrElse(profileSelector(state), undefined)
```

Comment:
I think we also have `toUndefined` in the pot library.

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1191/commits/1d38dc8d1a511d43e19b00d574aec0ca29a636c0